app/pages/sections/routes.test.js

```diff
@@ -661,10 +666,12 @@ describe('section routes', () => {
       expect(res.text.includes('data-test-id="error-title"')).toEqual(false);
     });
 
-    it('should return the correct status and text if response.success is true', async () => {
+    it('should return the correct status and text if response.success and api response.success is true', async () => {
```

Comment:
Change these to say `when the FE validation and the API call are both successful`